radio/src/telemetry/telemetry.cpp

```diff
@@ -381,7 +374,6 @@ void telemetryReset()
   telemetryData.hub.current = 55;
   telemetryData.hub.maxCurrent = 65;
 #endif
-#endif
 
 /*Add some default sensor values to the simulator*/
 #if defined(CPUARM) && defined(SIMU)
```

Comment:
I think this block can also be removed entirely.  I've been running it like this for some time now and haven't noticed any side effects.